### PR TITLE
Created Glue project

### PR DESCRIPTION
**Pull request by Motonstron**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: glue/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "lib": [
+        "es6"
+    ],
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": false,
```
Comment (plantain-00) [inline glue/tsconfig.json]: For new packages, this should be enabled

File: glue/tslint.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      // This package uses the Function type, and it will take effort to fix.
+      "ban-types": false
```
Comment (plantain-00) [inline glue/tslint.json]: for new packages, this file should only `"extends": "dtslint/dt.json"`, you can disable some rules in source code like `// tslint:disable-next-line ban-types`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

**Review by plantain-00 — APPROVED** [review]

**Comment by garfty** [comment]
Hi @minestarks my account is connected, not sure what happened but anything I can do to help I will 👍

**Comment by minestarks** [comment]
Oh hey, Travis is back! 

@garfty thanks, I'll merge once the tests pass.

**Comment by minestarks** [comment]
@garfty  uh oh, I missed this: All typings need to go under the `/types` path. Please don't add files to the root.  

I merged this already but would you mind opening a new PR moving these to the `types` subdirectory? They won't be published to `@types` as long as they're in here.

**Comment by garfty** [comment]
Hi @minestarks Apologies for this, I was wondering about it. I've now done this and a new PR is open [https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24500](here).

**Pull request merged**